packages/stylelint-config-terra/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Added $ to globals
```

Comment:
Since this package is disconnected from the WebDriver infrastructure I think we should include what we're adding the global for since it's just a magic wildcard here